<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClientView</class>
 <widget class="QWidget" name="TcpClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_ipaddr">
       <property name="text">
        <string>IP地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ipaddr"/>
     </item>
     <item>
      <widget class="QLabel" name="label_port">
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_port">
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_disconnect">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser_recv"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_send"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
